recipes/cryptopp/all/conandata.yml

```
@@ -4,8 +4,8 @@ sources:
       url: "https://github.com/weidai11/cryptopp/archive/CRYPTOPP_8_7_0.tar.gz"
       sha256: "8d6a4064b8e9f34cd3e838f5a12c40067ee7b95ee37d9173ec273cb0913e7ca2"
     cmake:
-      url: "https://github.com/abdes/cryptopp-cmake/archive/ff211af98505c260c1d3a36a8e36e563727b11ee.tar.gz"
-      sha256: "5a966e8a1af4aa39a4bc1f223562b4af97bd55385e9385b48b1f15fd0dc26b38"
+      url: "https://github.com/abdes/cryptopp-cmake/archive/4f52cc63b6e0bdd2a02a18745ba8984bcdad79e6.tar.gz"
+      sha256: "5430fa4de5d4d4c61a0172d95dbf33c2ef1845799691e017bed17933e4120fbd"
```

```suggestion
      url: "https://github.com/abdes/cryptopp-cmake/archive/51d00a28d761b7ae1585ee554a55fd8e6b943c9a.tar.gz"
      sha256: "94ded5059280eb550cd8ceed489db37faa4a2594a0c9a1910a2c848bfcc6abe7"
```

There is now a proper 8.7.0 version tag.